### DISCUSSION TITLE
HTML Parsing

### DISCUSSION THREAD

**Discussion by svonhatten** [cheeriojs/cheerio#2084] [Q&A] [answered]
## HTML Parsing

Hello! I am currently experiencing an issue with what is getting returned after implementing the `.load()` function, and was hoping to get some insight as to what could be going on.

I am passing a dom node to the `.load()` in this line of code: `htmlString = cheerio.load(domNode).html();` to get back the parsed html as a string.
Although I get a string of the HTML, the syntax is incorrect with regard to how the prop values are passed to their element tags. Instead of seeing `prop=value`, I am currently seeing `value:prop=value`. A real example would be the `lang` prop. Instead of `lang="en"`, I see `en:lang="en"`. Any insight or advice about how to fix this issue would be great, thank you!

As a side note, I am using `cheerio` version `"^1.0.0-rc.3"`.

**Comment by fb55** [comment, marked as answer]
Pretty sure this is an issue with serialization that's fixed in current releases. Can you try upgrading?

**Reply by svonhatten** [reply]
Ah I see! I will give that a go then, thank you!
